docs/faq/install.txt

```diff
@@ -50,12 +50,10 @@ What Python version can I use with Django?
 ============== ===============
 Django version Python versions
 ============== ===============
-3.2            3.6, 3.7, 3.8, 3.9, 3.10 (added in 3.2.9)
-4.0            3.8, 3.9, 3.10
-4.1            3.8, 3.9, 3.10, 3.11 (added in 4.1.3)
 4.2            3.8, 3.9, 3.10, 3.11, 3.12 (added in 4.2.8)
 5.0            3.10, 3.11, 3.12
 5.1            3.10, 3.11, 3.12
+5.2            3.10, 3.11, 3.12, 3.13
```

Comment:
@felixxm is it too early to say that it will support 3.13 or can we add because it is "planned" to support it?

Reply:
Having looked at some previous PRs, I think this is fine 👍

Reply:
It's fine